03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,4 @@
+1. Middleware is a function which we can evoke before we get the results for an API operation.
+2. Create, Read, Update, and Delete
+3.  a.process.env.KEY_NAME 
+    b.I add them in .gitignore file
```

Comment:
Right on point. Good work. If you get time, learn how the module - dotenv makes those in .env file available through process.env.variable (hint: config() function call).  It is always a good idea to use stackoverflow for any confusion to clarity. There are plenty of answers there for it in more detail. Stackoverflow is a go-to place for any question in my opinion.